static/css/emails.css

```diff
@@ -0,0 +1,261 @@
+/*
+Common CSS for emails
+ */
+body {
+    background-color: #1e4c59;
+    margin: 0;
+    padding: 0;
+}
+
+p {
+    margin: 0;
+}
+
+h1 {
+    margin: 0;
+}
+
+.main {
+    max-width: 600px;
+    margin: 0 auto;
+    text-align: center;
+}
+
+img.header {
+    max-width: 600px;
+    width: 100%;
+    display: block;
+}
+
+img.opencraft-logo {
+    margin: 32px 0;
+    width: 120px;
+    height: 57px;
+}
+
+.content {
+    background-color: white;
+    padding: 32px;
+    color: #0f1f24;
+}
+
+.addendum {
+    color: white;
+    font: 18px Chivo, sans-serif;
+    letter-spacing: 1px;
+    padding: 32px;
+    background-color: #6d9cae;
+}
+
+.addendum p {
+    max-width: 450px;
+    margin: 0 auto;
+}
+
+.footer {
+    font: 16px Chivo, sans-serif;
+    color: white;
+    padding: 32px 0;
+}
+
+.footer a {
+    color: white;
+    line-height: 1.31;
+}
+
+/*
+CSS for verification email
+*/
+
+.content h1 {
+    font: 24px 'Times New Roman', serif;
```

Comment:
@Agrendalath Ideally yes, but we don't have too much control over that since we're running an inlining script. It is because of this error that I changed to single quotes in CSS, where I usually use double quotes. But the inliner always produces this.